tests/functional/nullable-and-default-values-int/expected/data/out/db-dump/simple-int.csv

```diff
@@ -0,0 +1,4 @@
+"id","col-null-null","col-true-null","col-true-empty-string","col-true-string","col-true-zero-value","col-false-null","col-false-empty-string","col-false-string","col-false-zero-value"
```

Comment:
Nemam ti to kde okomentovat (prazdny subor), tak to komentujem tu.

Naco je `source/data/out/db-dump` ... v expected jasne, ... ale v source?

Reply:
jj to je blbost - je to pryč